### ISSUE TITLE
Parser breaks on `ORDER BY Timestamp`

### ISSUE THREAD

**Issue by recht** [AfterShip/clickhouse-sql-parser#87]
A query like this `SELECT Timestamp FROM events ORDER BY Timestamp` makes the parser panic:

```
panic: runtime error: invalid memory address or nil pointer dereference [recovered]
	panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x18 pc=0x101583b]

goroutine 19 [running]:
testing.tRunner.func1.2({0x1392880, 0x21fe600})
	external/go_sdk/src/testing/testing.go:1631 +0x24a
testing.tRunner.func1()
	external/go_sdk/src/testing/testing.go:1634 +0x377
panic({0x1392880?, 0x21fe600?})
	external/go_sdk/src/runtime/panic.go:770 +0x132
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseColumnExpr(0xc0000a2f10, 0x26)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:337 +0x7b
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseUnaryExpr(0xc0000a2f10, 0x26)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:304 +0x2c5
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseMulDivModExpr(0xc000151f10, 0xc00039bf80?)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:229 +0x1c
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseAddSubExpr(0xc000151f10, 0x7e4e636825b8?)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:182 +0x1c
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseCompareExpr(0xc000151f10, 0x26)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:123 +0x27
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseIsOrNotNull(0xc000151f10, 0x26)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:96 +0x3b
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseNotExpr(0xc0000a2f10, 0x1d)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:82 +0xe7
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseAndExpr(0xc000151f10, 0xc0000a2b40?)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:59 +0x1c
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseOrExpr(0xc000151f10, 0x3fc4f0?)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:37 +0x1c
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseExpr(0xc000151f10, 0xc0000a2c28?)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_column.go:16 +0x1c
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseOrderExpr(0xc000151f10, 0x1d)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_table.go:641 +0x25
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseOrderByClause(0xc000151f10, 0x1d)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_table.go:619 +0x7b
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).tryParseOrderByClause(0xc0000a2f10, 0x1d)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_table.go:612 +0x67
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseSelectStmt(0xc000151f10, 0x0)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_query.go:871 +0x479
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseSelectQuery(0xc000151f10, 0x1583bd5?)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_query.go:754 +0x159
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).parseStmt(0xc000151f10, 0x0)
	/home/joakim/go/pkg/mod/github.com/!after!ship/clickhouse-sql-parser@v0.3.4-0.20240627110054-61b3867d9915/parser/parser_table.go:939 +0x14d
github.com/AfterShip/clickhouse-sql-parser/parser.(*Parser).ParseStmts(0xc000151f10)
```

**Comment by git-hulk** [comment]
@recht Thanks for your report, it should be resolved in PR #88

**Comment by recht** [comment]
Awesome, thanks
